packages/mobile/src/screens/wallet-connect/types.ts

```diff
@@ -14,7 +19,11 @@ export type SignedMessageParams = {
 }
 
 export type WalletConnectParamList = {
-  Wallets: undefined | ConnectNewWalletParams | SignedMessageParams
+  Wallets:
```

Comment:
is there a reason for undefined vs `?:` here?

Reply:
yeah good q, i tihnk the undefined can be removed actually, i think the idea here is that params can be undefined, but they are always present? though in reality that means the same thing! will look into